src/main/java/fi/dy/masa/tweakeroo/renderer/RenderUtils.java

```diff
@@ -295,19 +294,21 @@ public static void renderDirectionsCursor(float zLevel, float partialTicks)
         int width = GuiUtils.getScaledWindowWidth();
         int height = GuiUtils.getScaledWindowHeight();
         Camera camera = mc.gameRenderer.getCamera();
```

Comment:
Matrix4fStack aware rendering for "F3 Cursor" rendering.  This code seems to work correctly.